GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -579,6 +579,11 @@ private void SetSelectedIndex(int index, bool toggleSelection = false)
             {
                 _gridView.Select();
 
+                if (index >= _gridView.Rows.Count)
+                {
+                    return;
+                }
```

Comment:
Can you please add a comment describing a situation that leads to this path?

Reply:
DOne.